products/devex/BUILD

```diff
@@ -215,13 +218,13 @@ pkg_tar(
 
 expand_workspace_status(
     name = "production-kustomization",
-    output = "products/devex/cd/overlays/production/kustomization.yaml",
-    template = "products/devex/cd/overlays/production/kustomization.tpl.yaml",
+    output = "cd/overlays/production/kustomization.yaml",
+    template = "cd/overlays/production/kustomization.tpl.yaml",
 )
 
 pkg_tar(
     name = "cd_production_patch",
-    srcs = ["products/devex/cd/overlays/production/kustomization.yaml"],
+    srcs = ["cd/overlays/production/kustomization.yaml"],
```

Comment:
why is this still required if we are deploying with `z app sync`? is this only for argo as it's reading the manifests from the zilliqa-manifests-internal. Maybe we can keep temporary this only for production till we migrate it

Reply:
Yes, it's not required. Keeping only for production as still using the old process. However, I just removed the block for Staging in a new commit.